src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheck.java

```diff
@@ -107,86 +107,189 @@
  * &lt;module name="ClassDataAbstractionCoupling"/&gt;
  * </pre>
  * <p>
- * To configure the check with a threshold of 5:
+ * Code examples:
+ * </p>
+ * <p>
+ * The check passes without violations in the following.
  * </p>
  * <pre>
- * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="max" value="5"/&gt;
- * &lt;/module&gt;
+ * class InputClassCoupling {
+ *   Set set = new HashSet(); // Ignored due to default excludedClasses property
+ *   Map map = new HashMap(); // Ignored due to default excludedClasses property
+ *   Date date = new Date(); // Counted, 1
+ *   Time time = new Time(); // Counted, 2
+ *   Place place = new Place(); // Counted, 3
+ * }
  * </pre>
  * <p>
- * To configure the check with two excluded classes {@code HashMap} and {@code HashSet}:
+ * The check results in a violation in the following.
  * </p>
  * <pre>
- * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="excludedClasses" value="HashMap, HashSet"/&gt;
- * &lt;/module&gt;
+ * class InputClassCoupling {
+ *   Set set = new HashSet(); // Ignored due to default excludedClasses property
+ *   Map map = new HashMap(); // Ignored due to default excludedClasses property
+ *   Date date = new Date(); // Counted, 1
+ *   Time time = new Time(); // Counted, 2
+ *   // instantiation of 5 other user defined classes
+ *   // Counted, 7
+ *   Place place = new Place(); // Violation
+ * }
  * </pre>
  * <p>
- * To configure the check with two regular expressions {@code ^Array.*} and {@code .*Exception$}:
+ * To configure the check with a threshold of 2:
  * </p>
  * <pre>
  * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="excludeClassesRegexps"
- *     value="^Array.*, .*Exception$"/&gt;
+ *   &lt;property name="max" value="2"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * The following example demonstrates usage of <b>excludedClasses</b> and
- * <b>excludeClassesRegexps</b> properties
+ * Code examples:
  * </p>
  * <p>
- * Expected result is one class {@code Date}
+ * The check passes without violations in the following.
  * </p>
  * <pre>
- * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="excludedClasses" value="ArrayList"/&gt;
- *   &lt;property name="excludeClassesRegexps" value="^Hash.*"/&gt;
- * &lt;/module&gt;
+ * class InputClassCoupling {
```

Comment:
Please make one code example for each config. Review and update other examples in the same way.

Reply:
Same confusion.

Reply:
Thanks for the link. Let is keep consistent.